.github/ISSUE_TEMPLATE/lsp_bug_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Language server client bug report
+description: Report a built-in lsp problem in Neovim
+labels: [bug, lsp]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report!
```

Comment:
> Thank you for taking the time to fill out this bug report!

I don't want to seem mean but I think we should be very efficient with our language here, so I would cut this.